### ISSUE TITLE
Conv1D dilation is not working.

### ISSUE THREAD

**Issue by KukhoHeo** [tensorflow/tflite-micro#1629]
Hi, im using tflite-micro latest version.
my specific version is follow:
Python:  3.7.9 (default, Aug 31 2020, 17:10:11) [MSC v.1916 64 bit (AMD64)]
Keras:  2.10.0
Tensorflow:  2.11.0-dev20220812

input_layer = Input(shape=(input_shape ))

#Series of temporal convolutional layers with dilations increasing by powers of 2.
conv_1 = Conv1D(filters=128, kernel_size=2, dilation_rate=2,
                padding='causal', strides=1,input_shape=input_shape,
                kernel_regularizer=regularizers.l2(0.01),
                activation='relu')(input_layer)

#Dropout layer after each 1D-convolutional layer
drop_1 = SpatialDropout1D(0.05)(conv_1)

#Flatten layer to feed into the output layer
flat = Flatten()(drop_1)

output_layer = Dense(2, activation='softmax')(flat)

TCN = Model(inputs=input_layer, outputs=output_layer)
TCN.compile(loss='mean_squared_error',
              optimizer=optimizers.Adam(lr=0.002),
              metrics=['mae', 'accuracy'])
TCN.fit(x_train, y_train,
          batch_size=128,
          epochs=10,
          verbose=1,
          validation_data=(x_test, y_test),
          callbacks = [TensorBoard(log_dir='../../logs/TCN1')])

When the following model is used, the values of the tflite result and the tflite-micro result are different.

but, if the dilation_rate set to 1, the results are same value.

So, i think dilation is not work on tflite-micro.

What should I do to get same result?
